packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -437,6 +437,7 @@ class RenderParagraph extends RenderBox with ContainerRenderObjectMixin<RenderBo
   @override
   void dispose() {
     _removeSelectionRegistrarSubscription();
+    _disposeSelectableFragments();
     // _lastSelectableFragments may hold references to this RenderParagraph.
     // Release them manually to avoid retain cycles.
     _lastSelectableFragments = null;
```

Comment:
I think we can remove this line and put `_disposeSelectableFragments` under the comment since it already nulls `_lastSelectableFragments`.

Reply:
Indeed. Thank you.

Reply:
I do not think the comment is relevant any more.